starboard/nplb/player_test_fixture.h

```diff
@@ -99,11 +101,15 @@ class SbPlayerTestFixture {
   void Write(const VideoSamples& video_samples);
   void Write(const AudioSamples& audio_samples,
              const VideoSamples& video_samples);
+
+  void Write(SbTime start, SbTime end);
```

Comment:
Consider removing the empty line above and add an empty line below, as this function is more close to the group above.